src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -168,13 +168,14 @@ internal Task FireReminderAsync(string actorTypeName, string actorId, string rem
         /// <param name="actorTypeName">Actor type name to invokde the method for.</param>
         /// <param name="actorId">Actor id for the actor for which method will be invoked.</param>
         /// <param name="timerName">The name of timer provided during registration.</param>
+        /// <param name="requestBodyStream">The name of timer provided during registration.</param>
```

Comment:
Bad copy-paste of docs. I'm generally in favor of removing doc comments on internal code. They end up being boilerplate. It's better to just use comments to call out the truly important details.